src/ResourceManagement/gulpfile.js

```diff
@@ -84,7 +84,7 @@ var codegen = function (project, cb) {
     const regenManager = args['regenerate-manager'] ? ' --regenerate-manager=true ' : '';
     var outputDir = path.resolve(mappings[project].dir);
     if (!args['preserve']) {
-        deleteFolderRecursive(outputDir);
+        deleteFolderRecursive(outputDir + '/Generated');
     }
```

Comment:
@RikkiGibson made a fix couple of days ago. Did you sinc with master?  here is the fix PR [https://github.com/Azure/azure-libraries-for-net/pull/162](https://github.com/Azure/azure-libraries-for-net/pull/162)